packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -538,9 +538,13 @@ contract Reserve is
       uint256 stableAmount;
       uint256 goldAmount;
       (stableAmount, goldAmount) = sortedOracles.medianRate(_tokens[i]);
-      uint256 stableTokenSupply = IERC20(_tokens[i]).totalSupply();
-      uint256 aStableTokenValueInGold = stableTokenSupply.mul(goldAmount).div(stableAmount);
-      stableTokensValueInGold = stableTokensValueInGold.add(aStableTokenValueInGold);
+
+      if (goldAmount != 0) {
```

Comment:
@m-chrzan Do you think I should add a check that the supply of the stabletoken should be zero as well?